streamz/core.py

```diff
@@ -319,18 +339,35 @@ def remove_stream(change):
 
         return output._ipython_display_(**kwargs)
 
-    def _emit(self, x):
+    def _emit(self, x, metadata=None):
```

Comment:
The use and structure of metadata, with particular reference to the reference counter, should be included here and for `emit()`

Reply:
We are talking about documentation?

Reply:
Yes please

Reply:
done.